x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/fields_browser.cy.ts

```diff
@@ -49,26 +47,15 @@ const defaultHeaders = [
   { id: 'user.name' },
 ];
 
-// Flaky in serverless tests
-// FLAKY: https://github.com/elastic/kibana/issues/169363
-describe.skip('Fields Browser', { tags: ['@ess', '@serverless'] }, () => {
-  context('Fields Browser rendering', () => {
+describe('Fields Browser', { tags: ['@ess', '@serverless'] }, () => {
+  describe('Fields Browser rendering', () => {
     beforeEach(() => {
```

Comment:
any reason to have both `beforeEach` exactly the same instead of having a single `beforeEach` in the top level `describe`?